src/compiler/csharp_generator.cc

```diff
@@ -49,7 +48,9 @@ using grpc::protobuf::Descriptor;
 using grpc::protobuf::ServiceDescriptor;
 using grpc::protobuf::MethodDescriptor;
 using grpc::protobuf::io::Printer;
-using grpc::protobuf::io::StringOutputStream;
+using grpc_generator::Printer;
```

Comment:
how exactly are you planning to switch out the protobuf dependency and provide an flatbuffer-specific implementation of these?  It seems like ripping out individual .cc files from our codebase and trying to get them built elsewhere.

Reply:
To be honest, maintaining this in a working condition sounds like more work that just rewriting the codegen for grpc-flatbuffers C# from scratch (which isn't really that difficult, its a few hunded lines of code that fprintf stuff and the logic is rather trivial).  I'm happy to be convinced otherwise :-)